11a-plots-examples-scatter.Rmd

```diff
@@ -0,0 +1,180 @@
+Using `stat_smoooth()` is a handy shortcut to achieve what we wanted. We should take some care to verify the results of this function are inline with what we expect. We can achieve this by manually running an `lm()`.
+
+```{r message = FALSE, warning = FALSE, error = FALSE}
+coef <- all %>%
+  filter(TRTA == "Xanomeline High Dose") %>%
+  lm(data = ., AVAL ~ BASE) %>%
+  coefficients()
+
+coef
+```
+
+The next step is to see if this is congruent with what `stat_smooth()` provided in our plot. We'll plot the coefficients and check whether they overlap, which would indicate they are similar.
+```{r message = FALSE, warning = FALSE, error = FALSE}
+all %>%
+  filter(TRTA == "Xanomeline High Dose") %>%
+  ggplot(., aes(x = BASE, y = AVAL)) +
+  geom_point() +
+  stat_smooth(method = "lm", se = FALSE, size = 5) +
+  geom_abline(intercept = coef[[1]], slope = coef[[2]], color = "red")
+```
+
+Looks okay!
+
+#### More faithful aesthetics
+While the above plot matched the technical specification, it did differ in a few aesthetics namely colors, and the border of the main plot. Below is code to achieve a closer aesthetic match with what's in the SSTIF for the interested.
+
+Line spaces have been removed from the beginning and end of text.
+```{r message = FALSE, warning = FALSE, error = FALSE}
+# setup plot text
+yaxis_text_alt <- "Maximum Post-baseline Systolic Blood Pressure (mmHg)"
+xaxis_text_alt <- "Baseline Systolic Blood Pressure (mmHg)"
+header_alt <- "Figure 17. Baseline vs. Maximum Systolic Blood Pressure by Treatment Arm, Safety Population¹, Pooled Analysis"
+footer1_alt <- "Source: [include Applicant source, datasets and/or software tools used]."
+footer2_alt <- "¹Gray dotted line = no increase; blue line = high-dose treatment linear regression; grey dashed line = low-dose linear regression."
+```
+
+Colors and border contexts have changed. Title/footers have been re-factored using `patchwork::plot_annotation()`
+```{r message = FALSE, warning = FALSE, error = FALSE, fig.fullwidth=TRUE, fig.width=12, fig.height=10}
+f17_alt <- ggplot(all, aes(x = BASE, y = AVAL, linetype = TRTA)) +
+  geom_point(colour = "black", shape = 21, size = 4, alpha = 0.6, aes(fill = TRTA)) +
+  stat_smooth(method = "lm", se = FALSE, aes(color = TRTA)) +
+  scale_color_manual(values = c("skyblue", "grey")) +
+  scale_fill_manual(values = c("skyblue", "white")) +
+  geom_abline(intercept = 0, slope = 1, size = 1, lty = "dotted", color = "#5A5A5A") +
+  scale_x_continuous(breaks = seq(90, 210, 30), limits = c(90, 210)) +
+  scale_y_continuous(breaks = seq(90, 210, 30), limits = c(90, 210)) +
+  labs(
+    y = yaxis_text_alt,
+    x = xaxis_text_alt
+  ) +
+  theme_bw() +
+  theme(
+    plot.background = element_rect(fill = NA, color = "skyblue3", size = 2, linetype = "solid"),
+    legend.position = "bottom",
+    legend.title = element_blank(),
+    axis.text = element_text(face = "bold")
+  ) 
+
+# plot body + title/captions as separate entities
+library(patchwork)
```

Comment:
one sentence to mention what {patchwork} is?

Reply:
hey @iqis thanks for the quick review. I'm thinking since composing plots through custom layouts will pop up over and over again, we can instead dedicate a section to this in Chapter 6 (plotting basics). That way any of the subsequent examples (e.g. swimmer, spider, waterfall, etc.) can be extended after knowing how to is introduced. 

what do you think?